scalyr_agent/agent_main.py

```diff
@@ -1207,6 +1207,17 @@ def __create_client(self, quiet=False):
         if self.__config.verify_server_certificate:
             ca_file = self.__config.ca_cert_path
             intermediate_certs_file = self.__config.intermediate_certs_path
+
+            # Validate provided CA cert file and intermediate cert file exists. If they don't
+            # exist, throw and fail early and loudly
+            if not os.path.isfile(ca_file):
+                raise ValueError(("Invalid path \"%s\" specified for the \"ca_cert_path\" config "
+                                  "option: file doesn't exist" % (ca_file)))
```

Comment:
There seems to be extra parens around the argument to `ValueError` and around `ca_file` as well.  Are those necessary?  Same down below.

Reply:
It's technically not required since a string which is split across multiple lines is already inside parenthesis, but it appears black formatting tool is not too opinionated about that and leaves it in place.